index.html

```diff
@@ -1063,6 +1063,10 @@ <h2><dfn>Predicted events</dfn></h2>
             <code>PointerEvent</code>s). If this event is a <code>pointermove</code> event, it is a sequence of
             <code>PointerEvent</code>s that the user agent predicts will follow the events in the
             <a>coalesced event list</a> in the future; otherwise it is an empty list.</p>
+            <div class="note">
+                <p>While <code>pointerrawmove</code> events may have a non-empty <a>coalesced event list</a>,
+                their <a>predicted event list</a> will, for performance reasons, always be an empty list.</p>
```

Comment:
I just want to err on the side of caution. This is a note. So theoritically if in the future we decide to add predicted events to the pointerrawmove it is still backward compatible. Right?
In otherwords, we shouldn't add a wpt test to assert the event.getPredictedEvents().length == 0?

Reply:
yes it's a non-normative not. i might even soften the language slightly here to not make it sound like an edict. and yes, normatively the `pointerrawmove` events must have a predicted events list, but the list can be empty. no hard requirement that it MUST be empty.